mllib/src/test/scala/org/apache/spark/mllib/regression/IsotonicRegressionSuite.scala

```diff
@@ -0,0 +1,216 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.mllib.regression
+
+import org.scalatest.{Matchers, FunSuite}
+
+import org.apache.spark.mllib.util.{LocalClusterSparkContext, MLlibTestSparkContext}
+import org.apache.spark.mllib.util.IsotonicDataGenerator._
+
+class IsotonicRegressionSuite
+  extends FunSuite
+  with MLlibTestSparkContext
+  with Matchers {
+
+  private def round(d: Double): Double =
+    Math.round(d * 100).toDouble / 100
+
+  test("increasing isotonic regression") {
+    val trainRDD = sc.parallelize(
+      generateIsotonicInput(
+        1, 2, 3, 3, 1, 6, 7, 8, 11, 9, 10, 12, 14, 15, 17, 16, 17, 18, 19, 20)).cache()
+
+    val alg = new IsotonicRegression
+    val model = alg.run(trainRDD, true)
+
+    model.labels should be(
```

Comment:
Unfortunately yes, `be()` requires the parentheses. Although scalatest matchers are used throughout spark codebase it is not used in mllib so I've changed to use `assert` instead.